src/Microsoft.DncEng.SecretManager/Readme.md

```diff
@@ -217,6 +217,13 @@ parameters:
   subscription: Azure subscription id
 ```
 
+#### Account Sas Token
+```yaml
+type: azure-storage-account-sas-token
```

Comment:
The name isn't quite accurate.  Since we are hard coding "blob" as the type, it's really a blob-access-token.  (And many of the permissions don't apply).

Should the "Services" also be parameterized to include table/queue as options?

Reply:
Which of the things did you do? :-)

Reply:
I added a service parameter.